src/frontend/src/env/networks.env.ts

```diff
@@ -96,3 +96,22 @@ export const BTC_TESTNET_NETWORK: Network = {
 export const BITCOIN_NETWORKS: Network[] = [BTC_MAINNET_NETWORK, BTC_TESTNET_NETWORK];
 
 export const BITCOIN_NETWORKS_IDS: symbol[] = BITCOIN_NETWORKS.map(({ id }) => id);
+
+export const CHAIN_FUSION_MAINNET_NETWORK_SYMBOL = 'Chain Fusion (all mainnets)';
```

Comment:
Should we use something like `ChainFustion` for the `symbol`?

I don't think we have the use case but, just in case we want to access the `symbol` as string comparison? Also also other symbol do not contains spaces.

e.g.

```
export const ETHEREUM_NETWORK_ID: unique symbol = Symbol('ETH');

export const ETHEREUM_NETWORK: EthereumNetwork = {
	id: ETHEREUM_NETWORK_ID,
	env: 'mainnet',
	name: 'Ethereum Mainnet',
	chainId: 1n,
	icon: eth,
	explorerUrl: ETHEREUM_EXPLORER_URL
};
```

Reply:
agreed!